hms-app/src/components/hackathon-details/NewHackathonDetails.tsx

```diff
@@ -50,17 +51,7 @@ const useStyles = createStyles((theme) => ({
 export default function NewHackathonDetails(props: IProps) {
   const { classes } = useStyles()
   const { hackathonID, type } = props
-  const [opened, setOpened] = useState({
-    hackathonModal: false,
-    participantsModal: false,
-    categoriesModal: false,
-    ideasModal: false,
-  })
-  const [deleteIDs, setDeleteIDs] = useState({
-    participantID: '',
-    categoryID: '',
-    ideaID: '',
-  })
+  const [opened, setOpened] = useState(false)
```

Comment:
it's not so obvious what `opened` means in this class.  I think it's referring to a modal window being displayed? maybe a more descriptive name would be helpful `modalOpened` or something along those lines